src/authentication/requestPasswordReset.ts

```diff
@@ -1,10 +1,21 @@
 import { createClientComponentClient } from "@supabase/auth-helpers-nextjs";
 import { logErrorReturnLogId } from "@/logger/logger";
+import { AuthError } from "@supabase/gotrue-js";
+import { helpEmailAddress } from "@/common/helpEmailAddress";
 
 interface RequestPasswordResetResponse {
     errorMessage: string | null;
 }
 
+function formatErrorMessage(error: AuthError): string {
+    let errorMessage = error.message;
+    // HTTP 429 is "too many requests". Usually means email rate limit has been exceeded.
+    if (error.status === 429) {
+        errorMessage += `.\nPlease contact admin for help.\n${helpEmailAddress}`;
```

Comment:
I think instead of having a separate variable it might make sense to just return here - e.g. if you use this method again to write a completely different message that doesn't use the submitted one. So something like
```
if (error.status === HttpStatusCode.TooManyRequests) {
   return error.message + ...
}
return error.message;
```
If there were multiple messages it might also be useful to have it as a switch statement and maybe put this function in a separate file, but I'm assuming a long list of messages here so it's up to you if you want to do these now or leave it for someone else when more cases are added

Reply:
I've done the return approach. I agree a switch would be good if we add more cases here but I'm not sure if we'll need to so left it for now.